core/PRP/Animation/plAnimPath.h

```diff
@@ -45,7 +45,7 @@ class PLASMA_DLL plAnimPath : public plCreatable {
    hsAffineParts fParts;
 
 public:
-    plAnimPath() : fController(NULL), fTMController(NULL) { }
+    plAnimPath() : fController(), fTMController(), fAnimPathFlags(), fMinDistSq(), fLength() { }
```

Comment:
These should be reordered to match declaration order (else GCC warnings)

Reply:
😨  what's GCC smokin'?